.github/workflows/test_all.yml

```diff
@@ -15,18 +15,18 @@ jobs:
     permissions:
       contents: write
       pull-requests: write
-  # call_github_app_test_target_org:
-  #   uses: ./.github/workflows/test_github_app_target_org.yml
-  #   secrets: inherit
-  #   permissions:
-  #     contents: write
-  #     pull-requests: write
-  # call_github_app_test:
-  #   uses: ./.github/workflows/test_github_app.yml
-  #   secrets: inherit
-  #   permissions:
-  #     contents: write
-  #     pull-requests: write
+  call_github_app_test_target_org:
+    uses: ./.github/workflows/test_github_app _target_org.yml
```

Comment:
@alexvanderberkel 
For testing purposes use here:

```yaml
uses: alexvanderberkel/actions-template-sync@feat/target-org-push
```

That way you have always the latest code within your branch.
It is not ready to be used for an automated test, but for your development purposes it should work.

For an automated test I have some ideas but not sure currently what makes sense

Reply:
@AndreasAugustin 

Sorry for the delay in getting further! Apologies. 

Hm. 

I am still puzzled as I am unsure what the reason is:

  ::info::execute cmd hooks enabled
  hi, we are within the prepush phase
  maybe you want to add further changes and commits
  ::info::push changes
  error: src refspec origin does not match any